style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
```

Comment:
- [ ] [OPTIONAL] Good job so far! Your project opens only the same size in the mobile version, resulting in a white field on the right when expanded from 300 px to 800 px. My advice to you would be to ask you to fix this problem. Let's learn [additional information](https://youtu.be/yU7jJ3NbPdA).

![image](https://user-images.githubusercontent.com/63915024/171686002-1ec1d01a-6395-46ec-9e98-ea44580c7085.png)